src/constants/analytics.ts

```diff
@@ -221,6 +221,18 @@ export const AnalyticsEvents = unionize(
       amount?: number;
       validatorAddress?: string;
     }>(),
+    Error: ofType<{
+      location: string;
+      error: Error;
+      metadata?: any;
+    }>(),
+    RouteError: ofType<{
+      transferType: 'deposit' | 'withdrawal' | 'transferOut' | undefined;
```

Comment:
nit: `transferType?: 'deposit' | 'withdrawal' | 'transferOut'` // can we just use `TransferType`?

nits that below instead of `| undefined` we can just use `xcxc?: ...`

Reply:
`TransferType` is an abacus enum i believe, so it doesn't nicely translate into string literals 👀  - right now we use  `TransferType.name` or `TransferType.rawValue` so i thought this was nicer since i'm not sure how kotlin enums serialize over events.

can definitely take a stab at it tho 

can also try to replace `| undefined` with `xxxx?:` - i was under the impression that `| undefined ` was a more accurate type since the property is always there, it's just sometimes set to undefined 🤔 but will try and report back

Reply:
hmm not a fan of how the transferType is serialized - will be leaving as str literal for now but we can always revisit :D
<img width="195" alt="Screenshot 2024-06-26 at 8 09 27 PM" src="https://github.com/dydxprotocol/v4-web/assets/37092291/0a84c08d-4fa1-4b32-928b-a0426be67be0">